clients/cobol-lsp-vscode-extension/src/__tests__/commands/ClearCopybookCacheCommand.spec.ts

```diff
@@ -13,60 +13,59 @@
  */
 
 import * as fs from "fs-extra";
+import { readdirSync } from "fs-extra";
 import * as path from "path";
+import { posix } from "path";
```

Comment:
Why using `path.posix.join` rather than `path.join`?

Reply:
reverted to path.join